src/sqlancer/tidb/TiDBProvider.java

```diff
@@ -136,6 +138,22 @@ public void generateDatabase(TiDBGlobalState globalState) throws Exception {
                 throw new AssertionError(e);
             }
         }
+
+        if (globalState.getDbmsSpecificOptions().getTestOracleFactory().get(0)
```

Comment:
This only checks for one test oracle, while multiple ones can be passed, I think. Perhaps we can check all of them?

Reply:
CERT is impossible to be passed with other oracles. In TiDB, only `WHERE` and `HAVING` oracles can be composed.
If we enforce to compose CERT with other oracles, it is incorrect to apply these settings to other oracles.

Reply:
I think we can then validate that only one test oracle has been specified before executing this code?